analysis/model/src/main/kotlin/de/maibornwolff/codecharta/util/ResourceSearchHelper.kt

```diff
@@ -6,26 +6,42 @@ import java.nio.file.Paths
 class ResourceSearchHelper {
 
     companion object {
-        fun isResourcePresent(resourceName: String, searchToken: String, searchOperator: (String, String) -> Boolean,
+
+        /**
+         * Checks whether a resource contains one of the specified searchTokens. To check that, the given searchOperator is used.
+         *
+         */
+        fun isResourcePresent(resourceName: String, searchToken: List<String>, searchOperator: (String, List<String>) -> Boolean,
```

Comment:
The comment makes clear, that the method name is not able alone to express what it is doing.